app/jobs/request_verification_code_text_message_job.rb

```diff
@@ -0,0 +1,12 @@
+class RequestVerificationCodeTextMessageJob < ApplicationJob
+  def perform(phone_number:, locale:)
+    TextMessageVerificationCodeService.request_code(
+      phone_number: phone_number,
+      locale: locale
+    )
+  end
+
+  def priority
+    PRIORITY_HIGH
+  end
```

Comment:
tiny nitpick - could the two jobs `email_verification_code_job` and `request_verification_code_text_message_job` have similar naming pattern + structure?

i.e. move the `priority` method to the top + rename this file to `text_message_verification_job`? I'd like it to be obvious these are jobs that serve the same purpose